tests/deprecated_classes.py

```diff
@@ -975,6 +975,26 @@
         "airflow.providers.amazon.aws.operators.batch.AwsBatchOperator",
         "airflow.contrib.operators.awsbatch_operator.AWSBatchOperator",
     ),
+    (
+        "airflow.providers.amazon.aws.operators.dms.DmsCreateTaskOperator",
+        "airflow.providers.amazon.aws.operators.dms_create_task.DmsCreateTaskOperator",
+    ),
+    (
+        "airflow.providers.amazon.aws.operators.dms.DmsDeleteTaskOperator",
+        "airflow.providers.amazon.aws.operators.dms_delete_task.DmsDeleteTaskOperator",
+    ),
+    (
+        "airflow.providers.amazon.aws.operators.dms.DmsDescribeTasksOperator",
+        "airflow.providers.amazon.aws.operators.dms_describe_task.DmsDescribeTasksOperator",
```

Comment:
```suggestion
        "airflow.providers.amazon.aws.operators.dms_describe_tasks.DmsDescribeTasksOperator",
```

You have a wrong file name which causes the test to fail.